man/kernel-command-line.xml

```diff
@@ -571,6 +571,18 @@
         is set in <filename>/etc/hostname</filename>. Note that this does not bar later runtime changes to
         the hostname, it simply controls the initial hostname set during early boot.</para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>systemd.tty.<replaceable>tty</replaceable>.term=</varname></term>
+        <term><varname>systemd.tty.<replaceable>tty</replaceable>.rows=</varname></term>
+        <term><varname>systemd.tty.<replaceable>tty</replaceable>.columns=</varname></term>
```

Comment:
mm, should swap the nameing around?

i.e. one day it might make sense to add defaults for these fields. this would be really easy to extend if we had the order `systemd.tty.term.<device>=`, as we then could say `systemd.tty.term=` could be the default for all ttys. This is not so easy if we have the order `systemd.tty.<device>.term=`, as this would create an ambiguity.